packages/workflow/src/utils/auth-utils.ts

```diff
@@ -10,16 +10,7 @@
  */
 import * as decode from 'jwt-decode'
 import * as Hapi from '@hapi/hapi'
-import { hasScope } from '@opencrvs/commons/authentication'
-
-export enum USER_SCOPE {
-  DECLARE = 'declare',
-  VALIDATE = 'validate',
-  REGISTER = 'register',
-  CERTIFY = 'certify',
-  RECORD_SEARCH = 'recordsearch',
-  VERIFY = 'verify'
-}
+import { PlainToken } from '@opencrvs/commons'
```

Comment:
TokenWithBearer = explicit type for `Bearer abc`
PlainToken = explicit type for  `abc`